exampleCourse/questions/element/unitsInput/server.py

```diff
@@ -0,0 +1,8 @@
+import prairielearn as pl
+
+
+def generate(data):
+    ureg = pl.get_unit_registry()
```

Comment:
How fast is this in the cached case? Is there a reason we're explicitly choosing to make `pint` part of our public interface? Unless there's a good reason not to, I'd rather just tell instructors to do something like `f"{num_here} ft"`.

Reply:
The speed difference was significant enough that I couldn't make progress on writing the test question without setting this up first as part of the element (it almost times out on every grade call). This is actually why I had to set some custom options for the test fixture to avoid creating this registry too many times:

https://github.com/eliotwrobson/PrairieLearn/blob/8ec7cd581560f49f7a34b1945ebabfa80d2781ef/elements/pl-units-input/units_utils_test.py#L9-L11

I haven't actually timed it, but removing `scope="module"` from the above decorator and running tests locally, the slowdown is extremely noticeable.

Regarding making `pint` part of the public interface, the reasons are twofold. Primarily, now that it's added as a dependency, I think any question code can import + call from `pint`, so adding this helper function avoids people running into slowdown because of creating a unit registry without using a cache file (and not knowing where a writeable directory might be).

Second, `pint` actually has uses beyond just this parsing for question writing. In particular, it allows for "unitful" computations, where operating on quantities with units will be automatically expressed in derived units, see https://pint.readthedocs.io/en/stable/getting/tutorial.html#simplifying-units.